src/spaced/spaced/game/controllers/auto_controller.cpp

```diff
@@ -0,0 +1,55 @@
+#include <imgui.h>
+#include <spaced/game/controllers/auto_controller.hpp>
+#include <cmath>
+
+namespace spaced {
+using bave::NotNull;
+using bave::Ptr;
+using bave::Rect;
+using bave::Seconds;
+
+namespace {
+[[nodiscard]] auto nearest_target_y(std::span<NotNull<IDamageable*> const> targets, glm::vec2 const muzzle) -> std::optional<Rect<>> {
+	auto ret = std::optional<Rect<>>{};
```

Comment:
what is `ret` shorthand for? I see it everywhere. `return_value`?

Reply:
Yeah pretty much 😄